bin/gen_PKGBUILD.py

```diff
@@ -0,0 +1,44 @@
+import tomli
+
+with open("pyproject.toml", 'rb') as f:
+    pyproject = tomli.load(f)
+
+description = pyproject["project"]["description"]
+version = open('anesthetic/_version.py','r').readlines()[0].split("=")[1].strip().strip("'")
+url = pyproject["project"]["urls"]["Homepage"]
+pyproject["project"]["dependencies"]
+rel=2
+
+
+PKGBUILD = """# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/wjhandley)
```

Comment:
This **could** get desynchronised from the `secrets.AUR*`. I'd suggest reading these from the env and setting it appropriately.

Reply:
secrets.AUR gives [wjhandley] rather than Will Handley, so this is probably not worth the added complexity.